client/src/pages/Apply/Apply.js

```diff
@@ -477,17 +462,12 @@ const Apply = ({ navigation }) => {
               </View>
 
               <View style={styles.checkboxes}>
```

Comment:
I'd also recommend merging this branch with dev, and then using Jamie's custom `<RequiredField/>` component to mark all required fields. See the Create Lesson Drawer or Create Unit Drawer for examples on how to use the `<RequiredField/>` component.